OSINT section wiki

gjaskula99 opened the pull request:
OSINT part for PUTrequest_ wiki.

Added content:
* Main page covering legal aspects, terminology, dorking and OPSEC basics
* Maltego interface, data import and custom transforms basics
* List of popular data sources

gjaskula99 committed:
Main page, maltego and sources

gjaskula99 committed:
Further expansion

Sources list extanted. Corrected typos. Changelog added on the bottom of each page.

mattix1710 commented on mkdocs.yml:
Where are links to maltego_wiki and sources_wiki?
You can add them in manner similar to Forensics:
```
- OSINT:
    - Intro:
    - SECTION_TITLE:
        - some_topic
    - other_topic
    - third_topic
```

gjaskula99 replied:
Updated as requested in [1404e60](https://github.com/putrequest/PUTwiki/pull/1/commits/1404e6004ad0707cb4922a534982c2dfb49d889f)

mattix1710 reviewed (changes requested):
Overall nice, 

-  found some misconfiguration in mkdocs.yml
- move images to `assets/OSINT/<topic>` location
    - later: use them in manner `assets/OSINT/<topic>/<image_name>`

gjaskula99 committed:
Pics moved to assets/osint

Pictures are now referring relatively to assets. Links to maltego_wiki and sources_wiki added to mkdocs.yml.

mattix1710 reviewed (approved):
nice

mattix1710 committed:
Merge branch 'main' into osint_wiki
